docs/CONTRIBUTING-zh_TW.md

```diff
@@ -150,12 +150,12 @@
     ```
 
 
-### Alphabetical order
+### 依照字母排序
```

Comment:
```suggestion
### <a id="alphabetical-order"</a>依照字母排序
```

Now the link should work

Reply:
Thanks, I fixed it in the current commit.